docker-compose.yml

```diff
@@ -4,24 +4,14 @@ services:
 
   backend:
     build:
-      context: .
-      dockerfile: ./backend/Dockerfile
+      context: ./backend
     container_name: django_backend
-    volumes:
-      - ./backend/:/app
     ports:
       - "8000:8000"
-    command: python manage.py runserver 0.0.0.0:8000
 
   frontend:
     build:
-      context: .
-      dockerfile: ./frontend/Dockerfile
-    image: nuxt_dev
+      context: ./frontend
     container_name: nuxt_frontend
-    volumes:
-      - ./frontend/:/app
-      - ./frontend/:/node_modules
```

Comment:
`volumes` is for attaching an outside volume in the system to the container. The `COPY` command though can be used for getting files into the image, as it's been used already. 

So here this is attaching two outside volumes and I believe is the reason why others were having issues with nuxt not found!

In the previous version of the `Dockerfile`, the `./frontend` directory was being fully `COPY`d in. However as the `volumes` here are attached, they replace the existing `node_modules` that were generated when the dependencies were resolved. For a new contributor who has an empty `node_modules` (or does not have nuxt), the `volumes` is replacing with the empty directory. My suspicion is that the `docker compose` might have still been working for @andrewtavis as nuxt was likely already installed locally and the replacing `node_modules` would have it. Hopefully this makes sense, but that would be my guess.

Reply:
I was wondering if it was something having to do with me having something locally installed. Makes a lot of sense 😊